tests/test_input.py

```diff
@@ -0,0 +1,38 @@
+from pydicer.input.web import WebInput, download_and_extract_zip_file
+from pydicer.input.test import TestInput
+from pydicer.input.filesystem import FilesystemInput
+
+import os
+
+def test_input_valid_working_dir_():
+    valid_test_input = WebInput(data_url="")
+    # Assert path to DICOMs exists
+    assert os.path.isdir(valid_test_input.working_directory)
+
+    valid_filesystem_input = FilesystemInput()
+    # Assert path to DICOMs exists
+    assert os.path.isdir(valid_filesystem_input.working_directory)
+
+def test_input_invalid_working_dir_():
+    invalid_test_input = WebInput(working_directory="NOT_VALID_PATH", data_url="")
+    # Assert path to DICOMs does not exist
+    assert not os.path.isdir(invalid_test_input.working_directory)
+
+    invalid_filesystem_input = FilesystemInput(working_directory="NOT_VALID_PATH")
+    # Assert path to DICOMs does not exist
+    assert not os.path.isdir(invalid_filesystem_input.working_directory)
+
+# Uncomment to test the TestInput class, will download zenodo zipfile and run tests!
```

Comment:
Any reason why this is commented?

I think the github pipeline should start downloading this file to run tests. We should just make sure that if the test data has already been downloaded once, it isn't downloaded again if its used in multiple tests.

Reply:
But we can discuss how to do that in the next meeting..

Reply:
I commented it out for when we want to run tests locally and not have to download the data every time. Happy to redirect this particular test to a different directory and have vsc not run it in pytest extension or just uncomment it completely

Reply:
I think you want to still run those tests locally. The key is to only have to download the test data once... So it would need to be stored somewhere where it isn't deleted, and won't redownload if it's already there. Probably into a directory in the repo folder, but make sure it's added to gitignore (or might already be excluded thanks to the current rules).